tracks/perl5/two-fer/mentoring.md

```diff
@@ -0,0 +1,102 @@
+# Mentoring
+
+This is a great opportunity to prepare for the track with attention to style conventions.
+This is currently the first reviewed problem on the Python track, and serves to
+make sure the student understands unit testing and Test Driven Design.
+
+## Reasonable Solutions
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+sub two_fer {
+    my $name = shift || "you";
+    return "One for $name, one for me.";
+}
+1;
+```
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+sub two_fer {
+    return "One for " . (shift || "you") . ", one for me.";
+}
+1;
+```
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+use experimental qw(signatures);
+sub two_fer ($name = 'you') {
+    return "One for $name, one for me.";
+}
+1;```
+## Common Suggestions
+- Suggest using a default value instead of any form of conditionals.
+- If the student is using perl 5.2>, suggest using the `experimentals` module but make them aware that they're unlikely to find that particular patter in the wild.
+- If the student is using perl <5.2, suggest upgrading to get new features - but, again, make them aware that not every situation they might walk into is going to be running the latest version
+- Some people need help with running the tests.
+- Check indentation.
+- Printing vs. returning
+## Talking Points
+- Default values
+- String interpolation
+- Style preferences. This exercise is a good opportunity to talk about style conventions like indentation, parameter parenthesis in method declarations (if using `experimental`) and removing redundant comments.
+  Given the place in the curriculum it may be worth not going too deep and to not address points that are controversial or personal preference.
+- Unit Tests. For some students this will be an introduction to the notion of unit tests.
+  Make the case that these allow them to be productive while engaging with the problem.
+## Mentoring Notes
+A friendly standard answer about how this can be done in one line, and a 'hint: use a different default value' to get rid of the conditionals, will be all you need for maybe 90% of the submissions.
```

Comment:
```suggestion
  Make the case that these allow them to be productive while engaging with the problem.
  
## Mentoring Notes

A friendly standard answer about how this can be done in one line, and a 'hint: use a different default value' to get rid of the conditionals, will be all you need for maybe 90% of the submissions.
```

White space (in the form of a blank line) around the things.  Allows for visually differentiating while skimming to find, and easier reading.